installer/roles/local_docker/defaults/main.yml

```diff
@@ -10,12 +10,10 @@ rabbitmq_port: "5672"
 rabbitmq_user: "guest"
 rabbitmq_password: "guest"
 
-postgresql_version: "10"
+postgresql_version: "10-alpine"
```

Comment:
Hi and sorry for the slow review. Does this 'just work' if you had the old image running and re-ran a newer installer? I'd like to test this first, but I don't have time to simulate an upgrade right now, as I'm leaving for vacation after tomorrow. I would merge this today if you reverted to the default value and specify the image you want on your side when running the installer.

Reply:
done